a10_octavia/common/utils.py

```diff
@@ -80,3 +82,13 @@ def convert_to_rack_vthunder_conf(rack_list):
                                    '\'ip_address:partition\' entries: {}'
                                    .format(list(duplicates_list)))
     return rack_dict
+
+
+def check_ip_in_subnet_range(ip, cidr_string):
+    subnet_ip, cidr = cidr_string.split('/')
+    avail_hosts = 32 - int(cidr)
```

Comment:
```python
avail_hosts = (1 << (32 - int(cidr)))
netmask = socket.inet_ntoa(struct.pack('>I', (1 << 32) - avail_hosts))
```

32-cidr gives the bit to enable not the # of hosts available. This change makes `avail_hosts` a bit more clear